### ISSUE TITLE
Parts of conch do not adhere to current twisted style guides

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7529]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| Alex reported|
|-|-|
|Trac ID|trac#7529|
|Type|enhancement|
|Created|2014-06-26 15:52:13Z|
|Branch|https://github.com/twisted/twisted/tree/conch-style-7529|

Notably:

* Use of implicit relative imports
* Incorrect number of newlines
* use of `implements` instead of `implementer`

Marking as a Python3 issue since the first and third items impact porting.

<details><summary>Searchable metadata</summary>

```
trac-id__7529 7529
type__enhancement enhancement
reporter__Alex Alex
priority__normal normal
milestone__Python_3_x Python-3.x
branch__branches_conch_style_7529 branches/conch-style-7529
branch_author__lvh lvh
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1403797933711316 1403797933711316
changetime__1403864887922604 1403864887922604
version__None None
owner__lvh lvh

```
</details>

**Comment by twisted-trac** [comment]
|<img alt="Alex's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_1"></a>|Alex commented|
|-|-|

This is resolved by https://github.com/twisted/twisted/pull/30

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_2"></a>|@lvh commented|
|-|-|

(In [42812]) Branching to conch-style-7529.

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_3"></a>|@lvh set owner to @lvh|
|-|-|

LGTM! The failure appears to be this totally unrelated py3 FAIL: test_assertRaisesContextUnexpected (twisted.trial.test.test_assertions.TestSynchronousAssertions)

... which wasn't introduced nor affected by this patch.

Thanks for working on this, Alex!

**Comment by twisted-trac** [comment]
|[<img alt="lvh's avatar" src="https://avatars.githubusercontent.com/u/97816?s=50" width="50" height="50">](https://github.com/lvh)<a name="note_4"></a>|@lvh set status to `closed`|
|-|-|

(In [42823]) Merge conch-style-7529: Minor conch style fixes

Author: Alex
Reviewer: lvh
Fixes: [#7529](https://github.com/twisted/twisted/issues/7529)

This makes some minor style fixes to Conch.
